### ISSUE TITLE
Icons on primary screen on dual-head

### ISSUE THREAD

**Issue by samupl** [mate-desktop/caja#274]
Feature request: it would be nice to be able to choose on which desktop will caja render icons. Currently I'm using dual-head with primary monitor on the right, and my desktop icons are on the left screen.
##

**Comment by lucasmr** [comment]
Caja 1.24.0 still has this issue.

**Comment by rocchidavide** [comment]
Any news about this feature? Having desktop icons on the wrong display is quiet annoying :(
Thank you! :)

**Comment by sunwire** [comment]
Any progress?

**Comment by lukefromdc** [comment]
Can you move icons there manually?

**Comment by Jorilx** [comment]
@lukefromdc Yes it's possible to move them, but if you do a "Organize desktop by name" everything gets back to the wrong monitor. New icons get created on the wrong monitor, too.

**Comment by lukefromdc** [comment]
My guess is "organize desktop by name" must automatically put
icons on the primary monitor. Most users probably want them there,
but most !=all

**Comment by ruthan** [comment]
Links are dead, was actually someone successful in forcing to add new files icons to primary screen? I have 3x 27 and left edge of extended desktop is really far..
  Also any Windows / MacOS user is used to adding all new icons - downloaded files / launchers etc on primary monitor.. it makes sense..
  On Linux it seems that selecting primary monitor actually does almost nothing.. its not even synched with Login screen / greeter setting..
  BTW i found about that one monitor composition nvidia Xinerama line (`Option         "metamodes" "DP-4: nvidia-auto-select +2560+0, DP-0: nvidia-auto-select +0+35, DP-2: nvidia-auto-select +5120+0" is you hardcode it without autoselect doesnt matter `) is able to crash whole Xserver if you are using Manjaro, funny is that this line is often added by nvidia drivers installation scripts..
   Its almost 20 years, when im waiting for really working primary monitor settings on Linux desktop :( Its not miracle that  it has 1% of users market share :( 
 Its not rocket science:
  1) add new icons on selected monitor
  2) show all windows on all panels
  3) and forcing new apps and specially games to start on primary monitor...
   All of these have some dead issues threads, someone maybe made it working through some ugly scripts, but this need some checkbox in desktop settings.
